src/main/java/io/github/ust/mico/kafkafaasconnector/kafka/MicoCloudEventImpl.java

```diff
@@ -199,4 +201,8 @@ public void setUnknownExtensions(String key, String value) {
     public Optional<String> getDataRef() {
         return Optional.ofNullable(dataRef);
     }
+
+    public Optional<String> getSubject() {
```

Comment:
We should probably document these getters (or at least link to our cloud event documentation in the class javadoc)